network-api/networkapi/wagtailpages/pagemodels/research_hub/library_page.py

```diff
@@ -57,7 +62,7 @@ class ResearchLibraryPage(research_base.ResearchHubBasePage):
         SynchronizedField("search_image"),
     ]
 
-    def get_context(self, request: "http.HttpRequest") -> "template.Context":
+    def get_context(self, request: "http.HttpRequest") -> "django_template.Context":
```

Comment:
Was this an import conflict? If so, it must have been hard to track down :face_exhaling:

Reply:
Hi @jhonatan-lopes, yes it was! However, after importing `template` as `django_template` the error was resolved.